kafka_consumer/tests/python_client/script.py

```diff
@@ -27,7 +27,7 @@
 sasl_kerberos_keytab = ""
 sasl_kerberos_principal = ""
 sasl_kerberos_service_name = ""
-security_protocol = ""
+security_protocol = "PLAINTEXT"
```

Comment:
I'm guessing this is to avoid dealing with certificates, right? Might merit a comment explaining why we can't use the default value.

Reply:
plaintext is the default value, if we set it to empty or none, confluent-kafka will throw an error

Reply:
ah, so this never worked? or did the default change in the new version of confluent-kafka?

We can skip the comment, it sounds like.